hyacinth/session.py

```diff
@@ -435,6 +435,7 @@ def post_webhook(self, url, model, events, fields=None):
                     "events": events,
                     "model": model,
                     "url": url,
+                    "expires_at": expires_at,
```

Comment:
Hm good point. I need to test that actually! I'll get back to you after I do. 👍 💎

Reply:
So if you post a webhook with this as None, it will create a new webhook with their default expiration, which I believe is 3 days. Should we make this default 31 days unless otherwise specified in `hyacinth` itself? @amackera

Reply:
sounds good